src/extension-analytics.js

```diff
@@ -31,16 +31,17 @@ import {triggerAnalyticsEvent} from './analytics';
  * @param {!Element} parentElement
  * @param {!JsonObject} config
  * @param {boolean=} loadAnalytics
+ * @param {boolean=} disableImmediate
  * @return {!Element} created analytics element
  */
 export function insertAnalyticsElement(
-  parentElement, config, loadAnalytics = false) {
+  parentElement, config, loadAnalytics = false, disableImmediate = false) {
   const doc = /** @type {!Document} */ (parentElement.ownerDocument);
   const analyticsElem = createElementWithAttributes(
       doc,
       'amp-analytics', dict({
         'sandbox': 'true',
-        'trigger': 'immediate',
+        'trigger': disableImmediate ? '' : 'immediate',
```

Comment:
I went to check how`<amp-analytics trigger=...>` would be interpreted, and I couldn't find it in the docs.  I expected it to be in https://www.ampproject.org/docs/reference/components/amp-analytics#specifying-configuration-data

Reply:
This is not something that can be set by the publisher as it causes validation to fail... its only allowed by the runtime.  Code pointer:  https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/amp-analytics.js#L161